unix/vncpasswd/vncpasswd.cxx

```diff
@@ -37,6 +37,15 @@
 
 #include <termios.h>
 
+#include <pwquality.h>
+#include <locale.h>
+#include <libintl.h>
+
+#define PACKAGE_NAME "tigervnc"
+#define PACKAGE_NAME2 "libpwquality"
+#define LOCALE_DIR  "/usr/share/locale"
```

Comment:
This should come from CMake. Please refer to vncviewer on how to get this.

Reply:
I was not very familiar with CMake before
Okay, I will make the necessary modifications according to your suggestions